File: source/code/vbnet/authentication/login_attempt_req.vb
Try
    Dim app = Await client.GetApplicationAsync(
        "https://api.stormpath.com/v1/applications/$YOUR_APPLICATION_ID")

    Dim loginResult = Await app.AuthenticateAccountAsync(
        "han@newrepublic.gov", "first2shoot!")

    Dim accountDetails = Await loginResult.GetAccountAsync()

Catch rex As ResourceException
    ' Bad login credentials!
    Console.WriteLine("Error logging in. " + rex.Message)
End Try

File: source/code/csharp/authentication/login_attempt_req.cs
try
{
    var app = await client.GetApplicationAsync(
        "https://api.stormpath.com/v1/applications/$YOUR_APPLICATION_ID");

    var loginResult = await app.AuthenticateAccountAsync(
        "han@newrepublic.gov", "first2shoot!");
        
    var accountDetails = await loginResult.GetAccountAsync();
}
catch (ResourceException rex)
{
    // Bad login credentials!
    Console.WriteLine("Error logging in. " + rex.Message);
}
